packages/caliper-fabric/test/sample-configs/NetworkConfigNotMutual.yaml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
I think we can name this explicitly `PeerGatewayNetworkConfigNotMutual` as it will only be used for testing the peer gateway